wcomponents-core/src/test/java/com/github/bordertech/wcomponents/WBeanContainer_Test.java

```diff
@@ -13,7 +13,7 @@
 
 	@Test
 	public void testChildAccessors() {
-		WBeanContainer container = new WBeanContainer();
```

Comment:
This should not really be changed as it is testing WBeanContainer.

Interestingly, there is no WContainer_Test. As this change is playing in this area, it might be worthwhile writing a WContainer_Test as well.

Reply:
Yep, makes sense. I reverted the changes in this file back to WBeanContainer.

Reply:
I added WContainer_Test.